client/my-sites/customer-home/cards/features/help-search/style.scss

```diff
@@ -128,17 +130,32 @@ $min_results_height: 180px;
 	}
 
 	.inline-help__results {
-		margin: #{$search_results_top_spacing + 1px} #{-$card_padding_small} 0;
 		min-height: $min_results_height;
-		padding: #{$search_results_top_spacing / 2 - 1px} 0 0 0;
-		border-top: 1px solid var( --color-border-subtle );
+		padding: #{$search_results_top_spacing - 1px} 0 0 0;
+		margin-left: -$card_padding_small;
+		margin-right: -$card_padding_small;
 
 		@include break-mobile {
 			margin-left: -$card_padding_large;
 			margin-right: -$card_padding_large;
 		}
 	}
 
+	.inline-help__results-title {
+		padding-left: $card_padding_small;
+		padding-right: $card_padding_small;
+
+		@include break-mobile {
+			padding-left: $card_padding_large;
+			padding-right: $card_padding_large;
+		}
```

Comment:
Future suggestion:

I don't think this was mentioned by Jeff, but the Site Setup card changes its internal padding to 32px at the `min-width: 1041px` breakpoint, if we added something similar here and to the learn-grow card, we could make the text within the cards aligned on large screens as well.

<img width="314" alt="Screen Shot 2021-06-18 at 4 58 16 PM" src="https://user-images.githubusercontent.com/22446385/122508920-e56fb400-d056-11eb-8725-1755c028e0e3.png">

Reply:
I think this is worth doing. I did tidy up some of this in another PR, but didn't notice the site setup checklist. I must have been testing on a site where setup was complete.
I think it makes sense to standardise on 24px for large screens

Reply:
Note to self: check what the padding looks like on a stats card that's filled with data

Reply:
yea the stats card looked good and also had 24px padding. 24px would be easier to standardize on because it's the default for `.card`